File: app/src/main/java/com/androidexamples/app/activity/CollapsingToolbarLayoutActivity.java
package com.androidexamples.app.activity;

import android.os.Bundle;

import com.androidexamples.app.R;

public class CollapsingToolbarLayoutActivity extends BaseActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_collapsing_toolbar_layout);
        loadComponents();
    }

    private void loadComponents() {
        setUpToolbar();
        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
    }
}

File: app/src/main/java/com/androidexamples/app/activity/CollapsingToolbarLayoutActivity.kt
package com.androidexamples.app.activity

import android.os.Bundle

import com.androidexamples.app.R

class CollapsingToolbarLayoutActivity : BaseActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_collapsing_toolbar_layout)
        loadComponents()
    }

    private fun loadComponents() {
        setUpToolbar()
        supportActionBar!!.setDisplayHomeAsUpEnabled(true)
    }
}
